userdatamodel/user.py

```diff
@@ -109,6 +115,18 @@ def __repr__(self):
         return self.__str__()
 
 
+class GoogleProxyGroup(Base):
+    __tablename__ = "google_proxy_group"
+
+    id = Column(String(90), primary_key=True)
```

Comment:
Just curious, is there a particular reason this is 90?

Reply:
Looks like it actually may be limited to 60 chars... https://support.google.com/a/answer/33386 . I suppose this could be enforced during the creation and this field could just be String and not rely on specific Google requirements?

Reply:
"Group names can have a maximum of 60 characters."

Reply:
changed in f3c6dd1